sdk/eventhub/azure-eventhub/stress/azure_eventhub_consumer_stress_async.py

```diff
@@ -89,6 +91,8 @@ def parse_starting_position(args):
 recv_cnt_iteration_map = defaultdict(int)
 recv_time_map = dict()
 
+azure_metric_monitor = AzureMonitorMetric("ASync EventHubConsumerClient", "ASync EventHubConsumerClient")
```

Comment:
```suggestion
azure_metric_monitor = AzureMonitorMetric("Async EventHubConsumerClient", "Async EventHubConsumerClient")
```
do we need both name and description here? if yes, they should be different - i'd prefer to keep description optional

Reply:
thanks for the suggestions! I thought it was mandatory because it's not optional in methods of opencensus.stats like `MeasureInt`, but apparently None is also allowed.

description is optional now